test/e2e/tests/new-folder-flow/new-folder-flow-empty.test.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { FolderTemplate } from '../../infra';
+import { test, expect, tags } from '../_test.setup';
+import { addRandomNumSuffix, createNewFolder, verifyFolderCreation } from './helpers/new-folder-flow.js';
+
+test.use({
+	suiteId: __filename
+});
+
+test.describe('Empty Folder - New Folder Flow', { tag: [tags.MODAL, tags.NEW_FOLDER_FLOW, tags.WEB] }, () => {
+	const folderTemplate = FolderTemplate.EMPTY_PROJECT;
+
+	test('Empty Folder - Folder Defaults', { tag: [tags.CRITICAL, tags.WIN] }, async function ({ app }) {
+		const folderName = addRandomNumSuffix('empty-project');
+
+		// Create a new empty project folder
+		await createNewFolder(app, {
+			folderTemplate,
+			folderName
+		});
+
+		await verifyFolderCreation(app, folderName, folderTemplate);
+	});
+
+	test('Only Empty Project template shows when interpreter startup behavior is disabled', async function ({ app, workspaceSettings }) {
+		// Disable startup behavior for all interpreters
+		await app.workbench.settings.removeWorkspaceSettings(['interpreters.startupBehavior']);
+		await workspaceSettings.set([['interpreters.startupBehavior', '"disabled"']]);
```

Comment:
Should we open an issue for updating the testing infra to include the ability to set language-scoped settings? It would allow us to add tests for R and Python/Jupyter when `interpreters.startupBehavior` is set at the python and r language scopes.

Reply:
Yes, that would be good.  Right now you can get one more deeply nested setting in using string:JSON where JSON is a string containing whatever JSON structure, but as soon as you try two of them, it would be broken because it wouldn't know they needed to merge and would make a broken JSON document.